jwst/associations/lib/rules_level2b.py

```diff
@@ -356,8 +356,8 @@ def __init__(self, *args, **kwargs):
                     Constraint(
                         [
                             DMSAttrConstraint(
-                                name='dithptin',
-                                sources=['dithptin'],
+                                name='',
```

Comment:
The previous version had the string "dithptin" populating both "name" and "sources", but now "name" is blank. Is that correct and/or desired?